src/ServiceInsight.Tests/MessageBodyViewModelTests.cs

```diff
@@ -31,15 +30,14 @@ public class MessageBodyViewModelTests
         [SetUp]
         public void TestInitialize()
         {
-            eventAggregator = Substitute.For<IEventAggregator>();
             workNotifier = Substitute.For<IWorkNotifier>();
             serviceControl = Substitute.For<IServiceControl>();
             hexContent = Substitute.For<HexContentViewModel>();
             jsonContent = Substitute.For<JsonMessageViewModel>();
             xmlContent = Substitute.For<XmlMessageViewModel>();
-            selection = new MessageSelectionContext(eventAggregator);
+            selection = new MessageSelectionContext(Substitute.For<IEventAggregator>());
 
-            messageBodyFunc = () => new MessageBodyViewModel(hexContent, jsonContent, xmlContent, serviceControl, eventAggregator, workNotifier, selection);
+            messageBodyFunc = () => new MessageBodyViewModel(hexContent, jsonContent, xmlContent, serviceControl, workNotifier, selection);
```

Comment:
I'm not sure that squashing all these commits into one is best. The refactorings each touch very disconnected parts of the code base so I'd prefer to keep them separate.

This can be fixed up by adding two fixup! commits for each of the offending commits.